hasura/metadata/actions.graphql

```diff
@@ -12,6 +19,17 @@ input create_circle_input {
   protocol_id: Int
 }
 
+input createUserInput {
+  address: String!
```

Comment:
We shouldn't allow the user to specify the address in the mutation. Instead we should set the ETH address of the authenticated user in the auth handler as `X-Hasura-Eth-Address` and extract the value in this action handler. That way its impossible to create a user with someone elses Eth address.